bulbs/contributions/views.py

```diff
@@ -165,6 +165,7 @@ def list(self, request):
         format = self.request.QUERY_PARAMS.get('format', None)
         if format == 'csv':
             queryset = self.get_queryset()
+            queryset = ESPublishedFilterBackend().filter_queryset(request, queryset, None)
```

Comment:
Possibly? I would like to rewrite this implementation so we aren't separating anything about the csv & api responses until the end, let that happen somewhere else.

TODO added.